app/pb_kits/playbook/pb_date_picker/sass_partials/_day_styles.scss

```diff
@@ -0,0 +1,47 @@
+// Day and Range Styles
+.flatpickr-day {
+  user-select: none;
+  @media (hover: hover) {
+    &:hover {
+      transition-duration: $transition_short;
+      transition-timing-function: $easeIn;    
+      background-color: #F6FAFF;
+      border: none;
+      box-shadow: $shadow_deep;
+    }
+  }
+}
+.flatpickr-day.today {
+  border: none;
+  font-weight: $bold;
+  color: $charcoal;
+  &.selected {
+    color: $text_dk_default;
+  }
+}
+.flatpickr-day.selected, .flatpickr-day.startRange, .flatpickr-day.endRange {
+  border-radius: 50px;
+  background-color: $primary;
+  border-color: $primary;
+  box-shadow: $shadow_deep;
+  font-weight: $bold; 
+}
+.flatpickr-day.startRange {
+  box-shadow: 17px 0 0 #F7FBFF
```

Comment:
Can we use our shadow variables?

Reply:
These have to be manual values.  They are the colors generating the range highlighting.  I used our shadow mixins/variables for the day hovers.